docs/faq.rst

```diff
@@ -502,17 +503,18 @@ instances, such as malfunctioning storage hardware, additional repo
 corruption may occur. If you can't afford to lose the repo, it's strongly
 recommended that you perform repair on a copy of the repo.
 
-In other words, the warning is there to emphasize that |project_name| will:
-  - Perform automated routines that will modify your backup repository
-  - May not actually fix the problem you are experiencing
-  - In very rare cases, may further corrupt your repository
+In other words, the warning is there to emphasize that |project_name|:
+  - will perform automated routines that modify your backup repository.
+  - Might not actually fix the problem you are experiencing.
+  - might, in very rare cases, further corrupt your repository.
```

Comment:
not sure if the (partial) sentences in the list should start with upper or lowercase, but please at least be consistent.